test/Libraries/CoreNodesTests/CoreNodeModelsTests/EnumAsStringTests.cs

```diff
@@ -0,0 +1,116 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Xml;
+using Dynamo.Graph;
+using Moq;
+using NUnit.Framework;
+using ProtoCore.AST.AssociativeAST;
+using TestUINodes;
+
+namespace DSCoreNodesTests
+{
+    [TestFixture]
+    public class EnumAsStringTests
+    {
+        private EnumAsStringConcrete testNode;
+        private Mock<AssociativeNode> associativeNodeMock;
+
+        [SetUp]
+        public void TestsSetup()
+        {
+            testNode = new EnumAsStringConcrete();
+            associativeNodeMock = new Mock<AssociativeNode>(MockBehavior.Strict);
+            testNode.Name = "testNodeName";
+        }
+
+        [Test]
+        public void PopulateItemsCoreTest()
+        {
+            var response = testNode.PopulateItemsCore("");
+            Assert.AreEqual("Restore", response);
+            Assert.AreEqual(3, testNode.Items.Count);
+        }
+
+        [Test]
+        public void SelectedIndexTest()
+        {
+            Assert.AreEqual(-1, testNode.SelectedIndex);
+
+            testNode.SelectedIndex = 1;
+            Assert.AreEqual(1, testNode.SelectedIndex);
```

Comment:
Is this worth testing 🤔

Reply:
I believe it is as there is some logic on that property's set method. https://github.com/DynamoDS/Dynamo/blob/932f867304cc87146c510b804af0b2aef82cfdbc/src/Libraries/CoreNodeModels/DropDown.cs#L83-L99